web-admin/src/features/organizations/hibernating/OrganizationHibernating.svelte

```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
+  import OrganizationHibernatingForAdmins from "@rilldata/web-admin/features/organizations/hibernating/OrganizationHibernatingForAdmins.svelte";
+  import OrganizationHibernatingForViewers from "@rilldata/web-admin/features/organizations/hibernating/OrganizationHibernatingForViewers.svelte";
+  import OrganizationAccessControls from "@rilldata/web-admin/features/organizations/OrganizationAccessControls.svelte";
+
+  export let organization: string;
+</script>
+
+<OrganizationAccessControls {organization}>
+  <OrganizationHibernatingForAdmins {organization} slot="manage-org" />
+  <OrganizationHibernatingForViewers slot="read-projects" />
+</OrganizationAccessControls>
```

Comment:
Here too, it'd be better if we passed around `orgPermissions`, so we could show `OrganizationHibernatingForViewers` when `manage-org === false`.